packages/react-three-rapier/src/components/Physics.tsx

```diff
@@ -293,7 +298,22 @@ export interface PhysicsProps {
   predictionDistance?: number;
 
   /**
-   * The Error Reduction Parameter in between 0 and 1, is the proportion of the positional error to be corrected at each time step
+   * Minimum number of dynamic bodies in each active island
+   *
+   * @defaultValue 128
+   */
+  minIslandSize?: number;
+
+  /**
+   * Maximum number of substeps performed by the solver
+   *
+   * @defaultValue 1
+   */
+  maxCcdSubsteps?: number;
+
+  /**
+   * The Error Reduction Parameter in between 0 and 1, is the proportion of the positional error to be corrected at each time step.
+   *
    * @defaultValue 0.8
    */
   erp?: number;
```

Comment:
Seems the actual erp default might be 0.6?

https://github.com/dimforge/rapier/blob/da92e5c2837b27433286cf0dd9d887fd44dda254/src/dynamics/integration_parameters.rs#L197

Maybe the rapier.js jsdoc is just out of date

Reply:
I think leaving this default untouched for this release is ok though

Reply:
No stress! I should have checked the source integration_paramaters file earlier.

And I agree, `0.8` is good for now.